clang/lib/Tooling/DependencyScanning/ScanAndUpdateArgs.cpp

```diff
@@ -102,6 +102,7 @@ void tooling::dependencies::configureInvocationForCaching(
   case CachingInputKind::CachedCompilation: {
     FrontendOpts.Inputs.clear();
     FrontendOpts.CASInputFileCacheKey = std::move(InputID);
+    FrontendOpts.ModuleMapFiles.clear();
```

Comment:
Should we be clearing all the other settings that `CachingInputKind::IncludeTree` clears above?

Reply:
I think we should clear most of them, yes. I'd like to keep this PR focused on making basic prefix mapping working, and do the rest on separate PRs. Are you okay with leaving a FIXME for now?

Reply:
sgtm